### ISSUE TITLE
Concise summary points and ratings for specific legal notices

### ISSUE THREAD

**Issue by ravicodelabs** [tosdr/edit.tosdr.org#1182]
Are specific "ToS" or "Privacy Policy" or other legal notices, considered as units, out of scope here? E.g. I see "points" for instance are analyzed, and rankings are given to "services," but specific legal notices don't seem to be covered. 

For instance, generally when one downloads a new piece of software, it will have some legal notice with it. If the software is upgraded, then it will likely have some updated legal documents. Even popular websites such as social media websites change their terms from time to time. 

Hence, it would be useful, I think, to be able to have a summary of a particular document, and perhaps a rating as well. This can cover the situation, for example, of an application with historically good terms that has included a clause that diverges. It would also be helpful for people to use as a tool to decide whether to upgrade things in certain cases, perhaps. 

I can imagine something like a unique identifier being assigned to a particular legal notice. Then, someone can look up advice/notes on a that particular notice. It also seems as though folks are annotating documents anyway to gather points, so might be implemented easier by leveraging that work already, organized by notice in addition to the other ways (e.g. points). 

Thoughts? 

Thanks so much to the tosdr team as well for all the hard work!

**Comment by Deveroonie** [comment]
So giving a grade to a ToS the same way we do to a service?

**Comment by ravicodelabs** [comment]
I think a grade for a ToS (or other legal notice/document) would be great. Annotated explanations that are easy to read and summarize larger passages, e.g. being able to scroll through a legal and see such annotations alongside it, would also be very nice I think. And, unique identifiers to avoid ambiguity in which ToS (or other legal notice) the metadata/feedback is for. 

Consider for instance how many people in the world do software upgrades for common tools, such as the popular social media accounts. There are so many people that could potentially benefit from doing a quick look at TOS;DR to check out a much friendlier summarization of the notice. And, the grade is useful too, e.g. a lower grade may prompt someone to look at the annotations. 

Thanks a bunch for your response!

**Comment by ravicodelabs** [comment]
TL;DR I think versioned notices may be a nice thing to have at least as prominently as individual points.
